src/UserInterface.php

```diff
@@ -20,17 +20,23 @@ public function getIdentity() : string;
     /**
      * Get all user roles
      *
-     * @return Iterable
+     * @return string[]
```

Comment:
`string[]` is a legacy syntax for `array<int|string, string>`.

We return an `iterable` here, so the returned type should be `iterable<int|string, string>` to denote that we return an iterable containing strings